assets/src/edit-story/components/library/panes/shared/pillGroup/pill.js

```diff
@@ -13,49 +13,56 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { useCallback, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+
 /**
  * Internal dependencies
  */
-import useRovingTabIndex from '../common/useRovingTabIndex';
+import useRovingTabIndex from '../useRovingTabIndex';
 import { useKeyDownEffect } from '../../../../keyboard';
-import { pill } from './pill';
+
+export const PILL_HEIGHT = 36;
 
 const PillContainer = styled.button`
-  ${pill};
+  border: 1px solid transparent;
+  margin-right: 12px;
+  padding: 7px 16px 8px;
+  height: ${PILL_HEIGHT}px;
+  border-radius: 18px;
+  font-size: ${({ theme }) => theme.fonts.paragraph.small.size};
+  line-height: ${({ theme }) => theme.fonts.paragraph.small.lineHeight};
+  user-select: none;
   cursor: pointer;
   margin-bottom: 12px;
-  font-family: ${({ theme }) => theme.fonts.body2.family};
+  font-family: ${({ theme }) => theme.fonts.paragraph.small.family};
   border-color: ${({ theme, isSelected }) =>
-    isSelected ? theme.colors.accent.primary : theme.colors.fg.gray24};
-  background-color: transparent;
+    isSelected ? 'transparent' : theme.colors.fg.gray16};
+  background-color: ${({ theme, isSelected }) =>
+    isSelected ? rgba(theme.colors.bg.divider, 0.04) : 'transparent'};
   color: ${({ theme }) => theme.colors.fg.primary};
-  user-select: none;
   background-clip: padding-box;
   transition: opacity 0.2s;
   &.invisible {
     opacity: 0;
   }
 `;
 
-PillContainer.propTypes = {
-  isSelected: PropTypes.bool,
-};
-
-const CategoryPill = ({
+function Pill({
```

Comment:
A few comments about this component in general:

* I notice that it uses a few `data-*` attributes, that I think we can do without. 
* The key listeners don't belong in this scope. Both of them (roving tabindex and _down-to-expand_) should go in the parent component. Events will properly bubble up to the parent scope. Thus there's no need for the `setIsExpanded` or `isExpanded` props either on this one.
* What's with the `className`? Is it used for styling or?
* The only reason we're passing `index` here is to auto-select the first item if none is selected. We can do that by automatically selecting the first id in the parent `PillGroup` whenever the list content changes. And then remove `index` here completely - that would clean up both this and the parent component a bit.

Reply:
> We can do that by automatically selecting the first id in the parent PillGroup whenever the list content changes

Can you elaborate on what you meant by "selecting the first id"? If you meant setting `isSelected` to `true` in case of the first ID then this would mean styling of the pill changing which would be incorrect.